### ISSUE TITLE
Ability to delete Lists, Tags etc. 

### ISSUE THREAD

**Issue by ghost** [nilfoer/mangadb#8]
Also I can not add manga's to my list while doing set_standart_lists [2021 Reading Challenge] it does as [2021 Reading Challenge] as different lists.

**Comment by ghost** [comment]
I delete manually my own.

**Comment by ghost** [comment]
After updating the new version I cant get work mangadb 
I get integration errors.
```
00:31:42 - INFO -  * Running on http://127.0.0.1:7578/ (Press CTRL+C to quit)
00:31:48 - INFO - Migrating DB with version 4 to newest version 5!
00:31:48 - ERROR - Upgrading from version 4 to 5 failed! DB was rolled back!
00:31:48 - ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "flask\app.py", line 2292, in wsgi_app
  File "flask\app.py", line 1815, in full_dispatch_request
  File "flask\app.py", line 1718, in handle_user_exception
  File "flask\_compat.py", line 35, in reraise
  File "flask\app.py", line 1813, in full_dispatch_request
  File "flask\app.py", line 1799, in dispatch_request
  File "manga_db\webGUI\webGUI.py", line 121, in show_entries
  File "manga_db\webGUI\webGUI.py", line 112, in get_books
  File "manga_db\webGUI\mdb.py", line 25, in get_mdb
  File "manga_db\manga_db.py", line 121, in __init__
  File "manga_db\manga_db.py", line 748, in _load_or_create_sql_db
  File "manga_db\db\migrate.py", line 240, in upgrade_to_latest
  File "manga_db\db\migrate.py", line 178, in _upgrade_to_version
  File "C:\Users\fuchi\Desktop\mango\manga_db\db\migrations\0005_ensure_titlecas
e.py", line 72, in upgrade
    c.execute(f"UPDATE {bridge_table} SET {fk_name} = ? WHERE {fk_name} IN "
sqlite3.IntegrityError: UNIQUE constraint failed: BookArtist.book_id, BookArtist
.artist_id
00:31:48 - INFO - 127.0.0.1 - - [25/Feb/2021 00:31:48] "←[35m←[1mGET / HTTP/1.1←
[0m" 500 -

```

**Comment by nilfoer** [comment]
Which version are you using (standalone windows etc. and if you're using the source version which python version and so on)?

**Comment by ghost** [comment]
Standalone windows

**Comment by nilfoer** [comment]
Should be fixed with v0.25.1

**Comment by ghost** [comment]
Yes it works now, thanks!
